java/server/src/org/openqa/selenium/grid/distributor/redis/RedisDistributorFlags.java

```diff
@@ -0,0 +1,59 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.grid.distributor.redis;
+
+import static org.openqa.selenium.grid.config.StandardGridRoles.DISTRIBUTOR_ROLE;
+
+import com.google.auto.service.AutoService;
+
+import com.beust.jcommander.Parameter;
+
+import org.openqa.selenium.grid.config.ConfigValue;
+import org.openqa.selenium.grid.config.HasRoles;
+import org.openqa.selenium.grid.config.Role;
+
+import java.util.Collections;
+import java.util.Set;
+
+@AutoService(HasRoles.class)
+public class RedisDistributorFlags implements HasRoles {
```

Comment:
Aren't these the same flags used by `RedisBackedSessionMap`? Having two sets of things with the same parameters will make jcommander deeply unhappy.

Reply:
How can I use the ones defined in `RedisBackedSessionMap` here 🤔 @shs96c

Reply:
As discussed on slack, we will be keeping this as is since it is possible that session map and redis can talk to different redis instances.